justfile

```diff
@@ -39,10 +39,10 @@ deploy-astria-local: deploy-namespace deploy-celestia-local deploy-sequencer
 
 deploy-geth-local: deploy-geth deploy-faucet deploy-blockscout deploy-ingress-local
 
-deploy-all-local: create-control-plane deploy-ingress-controller wait-for-ingress-controller deploy-astria-local deploy-geth-local
+deploy-all-local: create-control-plane deploy-ingress-controller wait-for-ingress-controller deploy-astria-local deploy-geth-local wait-for-sequencer
```

Comment:
We should probably wait for sequencer after it deploys instead of before geth comes up, at least right now gossipnet only works when sequencer is up before conductor, because conductor knows how to connect to sequencer as bootnode but conductor is non deterministic.